packages/web3-common/src/types.ts

```diff
@@ -207,12 +207,6 @@ export interface SyncOutput {
 	readonly pulledStates?: bigint | number;
 }
 
-export enum PredefinedBlockNumbers {
-	LATEST = 'latest',
-	PENDING = 'pending',
-	EARLIEST = 'earliest',
-}
-
```

Comment:
Renamed and moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/src/types.ts#L25)